packages/remix-dev/cli/run.ts

```diff
@@ -426,13 +426,10 @@ export async function run(argv: string[] = process.argv.slice(2)) {
       break;
     }
     case "dev":
+    default: // `remix ./my-project` is shorthand for `remix dev ./my-project`
       if (!process.env.NODE_ENV) process.env.NODE_ENV = "development";
       if (flags.debug) inspector.open();
       await commands.dev(input[1], process.env.NODE_ENV);
       break;
```

Comment:
You can delete this last `break` now, since it's not necessary in the `default` case.

```suggestion
```